src/Android/Xamarin.Android/Samples/Map/GetElevationAtAPoint/readme.md

```diff
@@ -0,0 +1,36 @@
+# Get elevation at a point
+
+Get the elevation for a given point on a surface in a scene. 
+
+![](GetElevationAtAPoint.jpg)
+
+## Use case
+
+Knowing the elevation at a given point in a landscape can aid in navigation, planning and survey in the field.
+
+## How to use the sample
+
+Tap anywhere on the surface to get the elevation at that point. Elevation is reported in meters since the scene view is in WGS84.
+
+## How it works
+
+1. Create a SceneView and Scene with an imagery base map.
+2. Set an ArcGISTiledElevationService as the elevation source of the scene's base surface.
```

Comment:
I recommend putting the types (e.g. `ArcGISTiledElevationService`) in backticks here. Also, should be **basemap** in the line above. Affects each readme, but I'm only commenting once.